javascript/core/recipes-i/js-practice-number-tofixed.md

```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: http://www.codewars.com/kata/57256064856584bc47000611
 linkType: codewars
+
+aspects:
+  - workout
```

Comment:
This is the same material as the currency formatter, so it should also be `introduction`.